api/src/main/java/org/openmrs/ui/framework/UiFrameworkConstants.java

```diff
@@ -24,7 +24,8 @@
     public static final String GP_FORMATTER_TIME_FORMAT = "uiframework.formatter.timeFormat";
     public static final String GP_FORMATTER_JS_DATETIME_FORMAT = "uiframework.formatter.JSdateAndTimeFormat";
     public static final String GP_FORMATTER_JS_DATE_FORMAT = "uiframework.formatter.JSdateFormat";
-    
+    public final static String GP_HANDLE_TIMEZONES = "uiframework.handleTimezones";
```

Comment:
If anything, rather:
```java
public final static String GP_TIMEZONE_CONVERSIONS = "uiframework.timezone.conversions";
```
@ibacher, please help... :-)

Reply:
I will change to `uiframework.timezone.conversions` until we get a better name.